nncf/common/accuracy_aware_training/runner.py

```diff
@@ -20,16 +21,47 @@
 from typing import TypeVar
 from abc import ABC
 from abc import abstractmethod
+import io
+import os
+import os.path as osp
```

Comment:
Please move this imports to the top of the file to meet [NNCF styleguide](https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md) reqirements